adbe/adb_enhanced.py

```diff
@@ -1531,3 +1534,25 @@ def _error_if_min_version_less_than(min_acceptable_version, device_serial=None):
 def _is_emulator():
     qemu = get_adb_shell_property('ro.kernel.qemu')
     return qemu is not None and qemu.strip() == '1'
+
+
+def switch_screen(switch_type):
+    if switch_type == SCREEN_TOGGLE:
+        c, o, e = toggle_screen()
+    else:
```

Comment:
Nit: You might make an early exit here by 

```
if switch_type == SCREEN_TOGGLE:
        c, o, e = toggle_screen()
        return o
```

And then the `else` case won't require an indentation.

Reply:
I lost an error message here, i was fixed in new commit